src/runtime_src/core/common/drv/include/xrt_cu.h

```diff
@@ -253,6 +253,9 @@ struct xrt_cu {
 	 * one for submit, one for complete
 	 */
 	struct task_struct	  *thread;
+	/* Good for debug */
+	u32			   sleep_cnt;
+	u32			   max_running;
```

Comment:
Avoid mixing tab and spaces...

Reply:
Thanks for the comment! But we use space after tab for alignment at many place. The checkpatch.pl allow tab followed by spaces.
No change at this time until we plan to clean up this coding style.